SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+## Best Practices
+
+~~REDACTED~~
+
+## Supported Versions
+
+~~REDACTED~~
+
+## Reporting a Vulnerability
+Through the hard work and dedication of our world class engineering department, we are certain that there are no, have never been any, and will never be any security vulnerabilities in FizzBuzzEnterpriseEdition©®™.
+
+## Approved development environments
+- Microsoft Notepad
+- Microsoft Office XP
+  - Microsoft Excel
+  - Microsoft Powerpoint
+  - Microsoft Word
+- ~~Microsoft Paint<sup>[1]</sup>~~
+
+macOS TextEdit is in stage 4 of the consideration process, but has not yet passed phase 2 of the brainstorming review pipeline.
```

Comment:
Let's write "MacOS" starting with capital letter, like everywhere else in project.

Reply:
> Let's write "MacOS" starting with capital letter, like everywhere else in project.

Why not Mac OS X?